src/web/theme/pages/OrderReference/OrderReference.scss

```diff
@@ -0,0 +1,35 @@
+.order-reference {
+  color: $fontSecondaryColor;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  &__title {
+    h1 {
+    }
+  }
+
+  &__action {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    width: 100%;
+    max-width: $desktop;
+
+    .button {
+      width: unset;
+    }
+
+    input {
+      border: none;
+      border-bottom: 1px solid gray;
+      margin: 2em auto;
+    }
+    &--error:after {
+      content: "the code provided was not correct or no products were linked to it";
```

Comment:
:laughing: It is not really a good idea to have actual content in your styles.

This should be in the DOM